web/src/components/data-fields/IndustryDropdown.tsx

```diff
@@ -55,9 +56,8 @@ export const IndustryDropdown = (props: Props): ReactElement => {
     let homeBasedBusiness: boolean | undefined = undefined;
     let cannabisLicenseType = undefined;
 
-    if (!isHomeBasedBusinessApplicable(industryId)) {
-      homeBasedBusiness = false;
-    }
+    homeBasedBusiness = isHomeBasedBusinessApplicable(industryId) ? homeBasedBusiness : false;
+    homeBasedBusiness = nexusLocationInNewJersey(state.profileData) ? false : homeBasedBusiness;
```

Comment:
[dust] Can we combine the logic for this variable? Would something like `homeBasedBusiness = isHomeBasedBusinessApplicable(industryId) && !nexusLocationInNewJersey(state.profileData)` work?